docs/Other SDL Features/Creating an App Service/index.md

```diff
@@ -746,3 +746,62 @@ sdlManager.addOnRPCRequestListener(FunctionID.PERFORM_APP_SERVICES_INTERACTION,
 });
 ```
 !@
+
+## Updating Your Published App Service
+Once you have published your app service, you may decide to update the type of data provided in your app service. For example, you may want to give all-access to paid subscribers but provide limited information to free users. You can do this by updating the app service manifest. If desired, you can also delete your app service by unpublishing the service. 
+
+### 7. Updating a Published App Service Manifest (RPC v6.0+)
+@![iOS]
+##### Objective-C
+```objc
+SDLAppServiceManifest *manifest = [[SDLAppServiceManifest alloc] initWithAppServiceType:SDLAppServiceTypeWeather];
+manifest.weatherServiceManifest = <#Updated weather service manifest#>
+
+SDLPublishAppService *publishServiceRequest = [[SDLPublishAppService alloc] initWithAppServiceManifest:manifest];
+[self.sdlManager sendRequest:publishServiceRequest];
+```
+
+##### Swift
+```swift
+let manifest = SDLAppServiceManifest(appServiceType: .weather)
+manifest.weatherServiceManifest = <#Updated weather service manifest#>
+
+let publishServiceRequest = SDLPublishAppService(appServiceManifest: manifest)
+sdlManager.send(publishServiceRequest)
+```
+!@
+
+@![android,javaSE,javaEE]
+##### Java
+```java
+AppServiceManifest manifest = new AppServiceManifest(AppServiceType.WEATHER);
+manifest.setWeatherServiceManifest("Updated weather service manifest");
+
+PublishAppService publishServiceRequest = new PublishAppService(manifest);
+sdlManager.sendRPC(publishServiceRequest);
+```
+!@
+
+### 8. Unpublishing a Published App Service Manifest (RPC v6.0+)
+@![iOS]
+##### Objective-C
+```objc
+SDLUnpublishAppService *unpublishAppService = [[SDLUnpublishAppService alloc] initWithServiceID:@"<#The serviceID of the service to unpublish#>"];
+[self.sdlManager sendRequest:unpublishAppService];
+```
+
+##### Swift
+```swift
+let unpublishAppService = SDLUnpublishAppService(serviceID: "<#The serviceID of the service to unpublish#>")
+sdlManager.send(unpublishAppService)
+```
+!@
+
+@![android,javaSE,javaEE]
+##### Java
```

Comment:
See above

Reply:
Fixed. Removed all Java headers for the code snippets